rebar.config

```diff
@@ -11,6 +11,7 @@
 {eunit_opts, [verbose]}.
 
 {deps, [
+<<<<<<< HEAD
```

Comment:
@korry8911 We have merge-conflict here.  You'll want the deps in the `HEAD` section except for the addition of the `rebar_lock_deps_plugin` we just added.  That needs to come from the `develop-2.2` side.